drivers/cc1200/gnrc_cc1200/gnrc_netdev_cc1200.c

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright (C) 2017 TU Berlin DAI Labor
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
```

Comment:
Add author

Reply:
The diff to [gnrc_netdev_cc110x.c](https://github.com/RIOT-OS/RIOT/blob/master/drivers/cc110x/gnrc_cc110x/gnrc_netdev_cc110x.c) is really small (except naming). This is an other aspect where I'd love to see some kind of unification between both implementations